apps/wpcom-block-editor/src/default/features/blogging-prompt.js

```diff
@@ -0,0 +1,18 @@
+import { createBlock } from '@wordpress/blocks';
+import { dispatch } from '@wordpress/data';
+import { getQueryArgs } from '@wordpress/url';
+import { isEditorReady } from '../../utils';
+
+const { answer_prompt } = getQueryArgs( window.location.href );
+
+if ( answer_prompt ) {
+	( async () => {
+		await isEditorReady();
+
+		const blockProps = answer_prompt ? { promptId: answer_prompt } : {};
```

Comment:
for later - this is unnecessary now that we took `new_prompt` out of this functionality to use with the modal instead. we can always use `promptId` with this value.